### ISSUE TITLE
Incorrect translating by using Gettext

### ISSUE THREAD

**Issue by sergeyklay** [phalcon/cphalcon#11311]
**How to reproduce**

1. Create `/tmp/test.php`
    ```php
    <?php
    _('Hello');
    _('Amazing %name%!');
    ```

2. Next run `xgettext /tmp/test.php` (this creates a file called `messages.po`)
3. Change the `"Content-Type: text/plain; charset=CHARSET\n"` in this file and replace CHARSET with `UTF-8` in this case. 
4. Change the `"Language: \n"` in this file: `"Language: uk\n"`
5. Next translate all strings:
    ```
    msgid "Hello"
    msgstr "Привіт"

    msgid "Amazing %name%！"
    msgstr "Чудовий %name%！"
    ```
6. Save and use the command `msgfmt -o messages.mo messages.po` which will create a compiled .mo file called `messages.mo`
7. Create locale dir `mkdir -p /tmp/locale/uk_UA.utf8/LC_MESSAGES`
8. Place this file in this directory `/tmp/locale/uk_UA.utf8/LC_MESSAGES/messages.mo`
9. Create test script `/tmp/gettext.php`:
```php
    <?php

    use Phalcon\Translate\Adapter\Gettext;

    $translator = new Gettext(
        [
            'locale'        => 'uk_UA.utf8',
            'defaultDomain' => 'messages',
            'directory'     => '/tmp/locale',
        ]
    );


    echo $translator->query('Hello'), PHP_EOL;
    echo $translator->t('Amazing %name%!', ['name' => 'Phalcon']), PHP_EOL;
    echo $translator->_('Hello'), PHP_EOL;
```

**Actual**
```
Hello
Amazing Phalcon!
Hello
```

**Expected**
```
Привіт
Чудовий Phalcon!
Привіт
```

**Comment by Jurigag** [comment]
Are you sure its phalcon problem ? You should check http://php.net/manual/en/function.gettext.php it with php builded functions.

**Comment by sergeyklay** [comment]
Fixed in `2.0.x`

**Comment by silviomedice** [comment]
No forget to set category
Set Category

    <?php

    use Phalcon\Translate\Adapter\Gettext;

    $translator = new Gettext(
        [
            'locale'        => 'uk_UA.utf8',
            'defaultDomain' => 'messages',
            'directory'     => '/tmp/locale',
           'category' => LC_MESSAGES
        ]
    );


    echo $translator->query('Hello'), PHP_EOL;
    echo $translator->t('Amazing %name%!', ['name' => 'Phalcon']), PHP_EOL;
    echo $translator->_('Hello'), PHP_EOL;
